pkg/kubelet/dockershim/remote/docker_server.go

```diff
@@ -72,7 +73,8 @@ func (s *DockerServer) Start() error {
 	runtimeapi.RegisterImageServiceServer(s.server, s.service)
 	go func() {
 		if err := s.server.Serve(l); err != nil {
-			klog.Fatalf("Failed to serve connections: %v", err)
+			klog.ErrorS(err, "Failed to serve connections")
+			os.Exit(255)
```

Comment:
```suggestion
			os.Exit(1)
```
https://github.com/kubernetes/community/pull/5486/files
Please add info in release notes.